src/Commands/CreateUserCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Commands;
+
+use App\Exceptions\User\CreateUserErrorException;
+use App\Repository\User\UserManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CreateUserCommand extends Command
+{
+    private const DEFAULT_PASSWORD = 'ToBeChanged';
+
+    private $userManager;
+
+    public function __construct(UserManagerInterface $userManager)
+    {
+        parent::__construct();
+
+        $this->userManager = $userManager;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('app.users.create')
+            ->setDescription('Create a new user into the CEM app')
+            ->setHelp('This command allows you to create a new user into the Community Event Manager. 
```

Comment:
Multiline content, please use a heredoc/nowdoc. Check whether you need to align left, or if we're 7.3 upward.